sonar-html-plugin/src/main/java/org/sonar/plugins/html/checks/comments/AvoidCommentedOutCodeCheck.java

```diff
@@ -40,6 +41,8 @@ public class AvoidCommentedOutCodeCheck extends AbstractPageCheck {
 
   private static final CodeRecognizer CODE_RECOGNIZER = new CodeRecognizer(THRESHOLD, LANGUAGE_FOOTPRINT);
 
+  private static final List<String> IGNORED_COMMENT_ANNOTATIONS = Arrays.asList("@thymesVar", "@elvariable");
```

Comment:
_minor_ since java 9, we usually use `List.of` when creating static final lists in classes, as it will return an immutable list.